src/lib/y2storage/storage_env.rb

```diff
@@ -89,6 +92,19 @@ def luks2_available?
       active?(ENV_LUKS2_AVAILABLE, default: false)
     end
 
+    # Whether YaST should reuse existing LVM
+    #
+    # see jsc#PED-6407 or jsc#IBM-1315
+    #
+    # @return [Boolean, nil] boolean as explicitly set by user, nil if user set nothing
+    def requested_lvm_reuse
+      value = read(ENV_REUSE_LVM)
+
+      return env_str_to_bool(value) if value
+
+      nil
```

Comment:
Nitpicking comment: I think this would do the same and it's nicer to my Ruby eyes

```ruby
def requested_lvm_reuse
  value = read(ENV_REUSE_LVM)
  return unless value

  env_str_to_bool(value)
```

How could you easily check if both implementations are equivalent? Well, I you would have unit tests... :wink: